R/target_sda.R

```diff
@@ -201,6 +203,12 @@ target_sda <- function(data,
   )
 }
 
+check_type_emission_factor <- function(ald) {
+  if (!is.double(ald$emission_factor)) {
+  abort("The column emission_factor of the asset-level data frame (`ald`) does not have the type double", class = "crucial_column_wrong_type")
```

Comment:
NIT: No need to write out the full name of the dataset. Since it is an argument of the function, referring to it simply as `ald` is sufficient detail.